Twig/TwigTemplate13.php

```diff
@@ -41,6 +41,12 @@ public static function attachRecorder(Recorder $recorder)
      */
     protected function getAttribute($object, $item, array $arguments = [], $type = \Twig_TemplateInterface::ANY_CALL, $isDefinedTest = false)
     {
+        @trigger_error(
+            'Contextual cache is broken since Twig v2.0. , and next major will not support Twig < 2.0. There is no '.
+            ' alternative solution provided yet.',
```

Comment:
When I said "linebreak", I was referring to the function and its arguments. When we linebreak error messages, we don't do it mid sentence, so that it stays somewhat greppable. See this thread: https://github.com/sonata-project/dev-kit/issues/218
@sonata-project/contributors we do linebreak them in CONTRIBUTING.md, but IIRC @Soullivaneuh is still against it, aren't you @Soullivaneuh? Anyway, I'm afraid we do not have an official decision about this somewhere, or is it really Solution 2 of https://github.com/sonata-project/dev-kit/issues/218 ?

Reply:
I just followed an example I found somewhere else in Sonata. I'd never split something like this over multiple lines (just information, not functional).
That being said: PR is updated :)

Reply:
> just information, not functional

Yeah once it's written, it's written, but it still has to be reviewed, and that's were things often go bad :P

Reply:
@greg0ire For now we still follow Symfony guidelines.